help.go

```diff
@@ -42,10 +42,10 @@ var helpSubcommand = &Command{
 }
 
 // Prints help for the App or Command
-type helpPrinter func(w io.Writer, templ string, data interface{})
+type helpPrinter func(w io.Writer, templ string, data interface{}, wrapAt int)
```

Comment:
It's not entirely clear from what's exported here, but making a change to the signature of these functions is unfortunately a breaking change, as the exported variables `HelpPrinter` and `HelpPrinterCustom` both use these signatures. [One of the CLIs I've written](https://github.com/rliebz/tusk/blob/e58ee3109f2f99972a35a054329787aa411c6357/appcli/help.go#L19-L22), for example, would no longer compile if this change were merged in.

I love the feature idea, but making a breaking change to implement it is a non-starter.

Reply:
Pushed an unpolished fixup which drops the `wrapAt` parameter, and instead allows the client to specify a "wrap" function to `HelpPrinterCustom`. If this seems reasonable I can add documentation, squash and rebase and perform some general cleanup etc.

Reply:
This seems to be going in the correct direction to me